### PR TITLE
Changed import numpy to import numpy as np

### PR DESCRIPTION
**Pull request by pgrinaway**
Ok, per #16, I've changed `import numpy` statements to `import numpy as np`. Let me know if I've missed anything...otherwise this should be relatively straightforward.

### CODE REVIEW

File: Yank/repex.py [hunk 1]
```diff
@@ -1340,9 +1339,9 @@ def _propagate_replica(self, replica_index):
         getstate_start_time = time.time()
         openmm_state = context.getState(getPositions=True)
         getstate_end_time = time.time()
-        self.replica_positions[replica_index] = openmm_state.getPositions(asNumpy=True)
```
Comment (kyleabeauchamp) [inline Yank/repex.py]: -1

File: Yank/repex.py [hunk 2]
```diff
@@ -1340,9 +1339,9 @@ def _propagate_replica(self, replica_index):
...
+        self.replica_positions[replica_index] = openmm_state.getPositions(asnp=True)
         # Store box vectors.
-        self.replica_box_vectors[replica_index] = openmm_state.getPeriodicBoxVectors(asNumpy=True)
```
Comment (kyleabeauchamp) [inline Yank/repex.py]: -1

File: Yank/repex.py [hunk 3]
```diff
@@ -1465,7 +1464,7 @@ def _minimize_replica(self, replica_index):
         minimized_positions = self.mm.LocalEnergyMinimizer.minimize(context, self.minimize_tolerance, self.minimize_maxIterations)
         # Store final positions
         openmm_state = context.getState(getPositions=True)
-        self.replica_positions[replica_index] = openmm_state.getPositions(asNumpy=True)
+        self.replica_positions[replica_index] = openmm_state.getPositions(asnp=True)
```
Comment (kyleabeauchamp) [inline Yank/repex.py]: -1

**Comment by kyleabeauchamp** [comment]
Looks reasonable once we fix the openmm copy paste issue and tests pass.

**Comment by pgrinaway** [comment]
Ok, hang on.

**Comment by kyleabeauchamp** [comment]
Merging now.  We should fix the OpenMMTools bug that's causing the segfault, though, as I think we want to get travis working soon.

**Comment by jchodera** [comment]
The segfault must be an openmm bug. We should try to produce a minimal case
for a bug report.

**Comment by kyleabeauchamp** [comment]
Given how long that will take to be released, IMHO we should first do a workaround in openmmtools.  Otherwise the yank release will have to wait until the *next* openmm release.

**Comment by jchodera** [comment]
> Given how long that will take to be released, IMHO we should first do a workaround in openmmtools. Otherwise the yank release will have to wait until the next openmm release.

Agreed.  I had tried to revert this by adding back `LennardJonesGrid`, but that didn't seem to fix the segfault. Will debug a bit more.

**Pull request merged**